src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -437,9 +437,9 @@ private function registerDoctrineMongoDbOdmConfiguration(ContainerBuilder $conta
         }
 
         $container->registerForAutoconfiguration(AggregationItemExtensionInterface::class)
-            ->addTag('api_platform.doctrine.mongodb.aggregation_extension.item');
+            ->addTag('api_platform.doctrine_mongodb.odm.aggregation_extension.item');
         $container->registerForAutoconfiguration(AggregationCollectionExtensionInterface::class)
-            ->addTag('api_platform.doctrine.mongodb.aggregation_extension.collection');
+            ->addTag('api_platform.doctrine_mongodb.odm.aggregation_extension.collection');
```

Comment:
many thanks about the fix @markitosgv ! Could you update the tests accordingly?

Reply:
sure ;-)